src/components/HeroSection/HeroSection.css

```diff
@@ -0,0 +1,37 @@
+.container {
+  background-position: center;
+  background-size: cover;
+  height: 55vh;
+  width: 100%;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.findmebutton {
```

Comment:
class names should be camelCase

Reply:
Of Course, I am on it. Fixing it. Thanks Halit